### DISCUSSION TITLE
Ory session to jwt implementation example

### DISCUSSION THREAD

**Discussion by tundak** [ory/.github#115] [Q&A]
Is there any example implementation available on self hosted setup for https://www.ory.sh/docs/identities/session-to-jwt-cors ?

**Comment by vinckr** [comment]
Hello @tundak 

Not directly, but it would work pretty much the same way as when using Ory Network.  
The difference is that you would not use the CLI command `ory patch identity-config` to set the JWKS and claims.jsonnet but instead edit your Ory Kratos configuration file (most likely named kratos.yaml) directly. You can still use the CLI to create the JWKS, but you don't need to.
